Classes/Settings.cs

```diff
@@ -490,5 +481,32 @@ protected static void SetValue(string propertyName, string value)
                 }
             }
         }
+
+        private static string GetValuePostfix(Model.Programme progInfo, string name, string defaultValue)
```

Comment:
Have you noticed that these functions are private? Because of progInfo they can't be internal. So overloads would too far a stretch.